### PR TITLE
[TESTING] Added test cases for CarServiceImpl

### PR DESCRIPTION
**Pull request by AntonZhdanov**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,8 +4,9 @@ services:
   mysqldb:
     platform: linux/x86_64
     image: mysql:8
+    container_name: car_sharing-db
     restart: unless-stopped
-    env_file: ./.env
+    env_file: .env.sample
```
Comment (ivan0dyatlyukkk) [inline docker-compose.yml]: Please, replace this custom option to the previous one

File: docker-compose.yml [hunk 2]
```diff
@@ -17,7 +18,7 @@ services:
     restart: on-failure
     image: car-sharing/docker-app
     build: .
-    env_file: ./.env
+    env_file: .env.sample
```
Comment (ivan0dyatlyukkk) [inline docker-compose.yml]: The same

**Review by ivan0dyatlyukkk — CHANGES_REQUESTED** [review]
GJ! But please, check my suggestions :)

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,4 @@
+.env.sample
```
Comment (OlegStashkiv) [inline .gitignore]: This should not be here. We are pushing this file

**Review by OlegStashkiv — CHANGES_REQUESTED** [review]
The tests themselves are ok. Just correct my and Ivan's remark

**Review by OlegStashkiv — APPROVED** [review]

**Review by ivan0dyatlyukkk — APPROVED** [review]

**Pull request merged**